sisl/viz/plotly/plots/pdos.py

```diff
@@ -23,6 +23,9 @@ class PdosPlot(Plot):
     tbt_nc: tbtncSileTBtrans, optional
         This parameter explicitly sets a .TBT.nc file. Otherwise, the PDOS
         file is attempted to read from the fdf file
+    geometry: str or Geometry or asciiSileBigDFT or cubeSile or gotSileGULP or hamiltonianSile or omxSileOpenMX or pdbSile or refSileScaleUp or restartSileScaleUp or tshsSileSiesta or onlysSileSiesta or ncSileSiesta or xvSileSiesta or fdfSileSiesta or outSileSiesta or pdosSileSiesta or deltancSileTBtrans or tbtsencSileTBtrans or phtsencSilePHtrans or tbtncSileTBtrans or tbtavncSileTBtrans or phtncSilePHtrans or phtavncSilePHtrans or tbtprojncSileTBtrans or phtprojncSilePHtrans or carSileVASP or chgSileVASP or locpotSileVASP or winSileWannier90 or xsfSile or axsfSile or xyzSile, optional
```

Comment:
perhas change to `Sile` instead of listing all possibilities?

Anything that has `read_geometry` would work?

Reply:
> Anything that has read_geometry would work?

Yes

This string is obviously generated automatically, but I will change it because I agree it doesn't make sense to write all of the possibilities.